rhoas-cli-service-registry/README.adoc

```diff
@@ -0,0 +1,374 @@
+////
```

Comment:
This is new service registry tutorial that bases on similar strategy that we have for kafka